mods/ts/sequences/structures.yaml

```
@@ -305,6 +305,11 @@ gapile:
 		Offset: 0, 0
 		DepthSprite:
 
+gapave:
+	icon: paveicon
```

```suggestion
	icon:
		Filename: paveicon
```
to match the new sequences setup after the rebase you did.